canton/BUILD.bazel

```diff
@@ -17,6 +17,7 @@ java_binary(
     runtime_deps = [
         # Do not sort
         "//canton:lib",
+        "@maven//:org_bouncycastle_bcprov_jdk15on",
```

Comment:
do you need this even though the bcprov jar is in `canton/lib` ? The risk here is that if we update the version in canton we forget to update it here and get mismatching runtime versions ?

Reply:
We can try to do something better in main, for 2.7.x this seems more than acceptable for me.